scripts/variantstore/scripts/run_avro_query_for_sample_info.py

```diff
@@ -59,10 +60,18 @@ def construct_sample_info_avro_queries(call_set_identifier, dataset_name, projec
     parser.add_argument('--dataset_name',type=str, help='BigQuery dataset name', required=True)
     parser.add_argument('--project_id', type=str, help='Google project for the GVS dataset', required=True)
     parser.add_argument('--avro_prefix', type=str, help='prefix for the Avro file path', required=True)
+    parser.add_argument('--new_sample_cutoff', type=str, help='cutoff for sample ids')
 
     args = parser.parse_args()
 
-    construct_sample_info_avro_queries(args.call_set_identifier,
+    if (args.new_sample_cutoff):
+            construct_sample_info_avro_queries(args.call_set_identifier,
+                           args.dataset_name,
+                           args.project_id,
+                           args.avro_prefix,
+                           args.new_sample_cutoff)
+    else:
```

Comment:
I don't think you need this if/else, you should be able to just pass that the `args.new_sample_cutoff` in all circumstances.